handler.go

```diff
@@ -24,6 +25,23 @@ type Request struct {
 	QueryString string
 	Method      string
 	Host        string
+
+	ctx context.Context
```

Comment:
Is this field normally encapsulated in other code, like the stdlib?

Reply:
Yes it is

Reply:
I can see why this is here now -> https://github.com/openfaas-incubator/go-function-sdk/pull/6#discussion_r402973742